VultisigApp/VultisigApp/Views/Components/TokenSelectorDropdown.swift

```diff
@@ -59,8 +59,13 @@ struct TokenSelectorDropdown: View {
     
     var image: some View {
         ZStack {
-            ImageView(selected.logo, size: CGSize(width: 32, height: 32))
-                .cornerRadius(100)
+            Group {
+                if selected.logo.hasPrefix("https://") {
```

Comment:
same

Reply:
Fixed, now we have only one component for images. All encapsulated.